### PR TITLE
feat(popover): close menu programmatically

### PR DESCRIPTION
**Pull request by ExeAy**
## **Describe pull-request**  
Added a `close()` method to the popover menu. This will make it possible to programmatically close the menu which is for example needed when having some button as a menu item that doesn't act as a link. 

## **Issue Linking:**  
- [**GitHub**](https://github.com/scania-digital-design-system/tegel/issues/709)

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Implement for example a button that calls the `close()` method provided by the popover menu.
2. Click the button and make sure that the menu closes.

## **Checklist before submission**
- [x] I have added unit tests for my changes (if applicable)
- [x] All existing tests pass
- [x] I have updated the documentation (if applicable)

## **Suggested test steps**
- [x] Browser testing (Chrome, Safari, Firefox) 
- [ ] Keyboard operability
- [ ] Design/controls/props is aligned with other components
    - Not exactly, I wanted an `open` prop instead to align with for example side menu dropdown. But the popover menu has a conflicting variable called `show` that has a slightly other purpose here.
- [x] Events

### CODE REVIEW

File: packages/core/src/components/popover-menu/popover-menu.stories.tsx [hunk 1]
```diff
@@ -117,24 +117,20 @@ const Template = ({ menuPosition, icons, fluidWidth }) => {
           <a href="#"> ${icons ? '<tds-icon name="share"></tds-icon>' : ''} Action </a>
         </tds-popover-menu-item>
         <tds-popover-menu-item disabled>
-          <button> ${icons ? '<tds-icon name="share"></tds-icon>' : ''} Action </button>
+          <button> ${icons ? '<tds-icon name="share"></tds-icon>' : ''} Disabled action </button>
```
Comment (theJohnnyMe) [inline packages/core/src/components/popover-menu/popover-menu.stories.tsx:120]: Nice catch!

**Review by theJohnnyMe — APPROVED** [review]
Thank you for a new feature @ExeAy !
Tested and it works nice. One thing we could do is to add the same feature to popover-canvas too.

**Comment by ExeAy** [comment]
> Thank you for a new feature @ExeAy ! Tested and it works nice. One thing we could do is to add the same feature to popover-canvas too.

Oh, for sure. I'll get to it later today or this week when I find time.

**Review by timrombergjakobsson — APPROVED** [review]
LGTM!

**Review by Lunkan89 — APPROVED** [review]
Nice work!

**Pull request merged**